src/common/Common.ts

```diff
@@ -110,14 +143,29 @@ export class Version {
         this.patch = patch;
     }
 
+    /**
+     * Verifies that the input version is an exact match.
```

Comment:
Oh this has nothing to do with Android. It's just a class that defines semantic versioning. And the way this class and its methods are defined, it would be used in cases like:

```
let v1 = new Version(....)
let v2 = new Version(....)
if (v1.same(v2)) {
. . . 
}
```

So methods like `same`,  `sameOrNewer`, etc will be invoked on an instance and take another instance. This is something common that developers are very well familiar with.